### DISCUSSION TITLE
Parameter Change with function instead of value

### DISCUSSION THREAD

**Discussion by dmilson14** [pybamm-team/PyBaMM#3821] [General]
I am currently attempting to do some lithium plating modelling using the O'Kane 2022 parameter set but with as many of the values from the Marquis 2019 set as allowed to look at lithium plating on this specific battery. When updating parameter from value to value I do not get an error but when attempting to update a parameter using a function, such as I have shown below, I get a whole heap of syntax errors. Is there any way to update a parameter value using a function?

Example: parameter_values.update({"Electrolyte conductivity [S.m-1]": <function electrolyte_conductivity_Capiglia1999 at 0x000001940E2A1DA0>})

^ This is to change the electrolyte conductivity from : <function electrolyte_conductivity_Nyman2008_arrhenius at 0x000001A7023A2160> which is the default one given by the O'Kane 2022 Parameter set.

**Comment by brosaplanella** [comment]
Not sure I fully understood your question, but if you just want to update a parameter function, take a look at this notebook: https://docs.pybamm.org/en/latest/source/examples/notebooks/getting_started/tutorial-4-setting-parameter-values.html#Function-parameters
